bin/rialto/runtime/src/lib.rs

```diff
@@ -416,6 +416,22 @@ impl pallet_substrate_bridge::Config for Runtime {
 	type BridgedChain = bp_millau::Millau;
 }
 
+parameter_types! {
+	// One hundred and fifty old headers is ~15 mins (assuming a target block time of 6s). This
+	// should give the relayer enough time to get a justification for a header.
```

Comment:
The justification time is not really up to the relayer. Restricting to 150 here means we assume that GRANDPA will produce at least one justification every 15 mins. I'd rather relax this and make it match the session length (GRANDPA has to provide at least one justification for each session).